### DISCUSSION TITLE
Failed to update an existing Edge Function's body

### DISCUSSION THREAD

**Discussion by WilliamScott-AdventGX** [supabase/supabase#7937] [Questions] [answered]
When executing the command `supabase functions deploy my-function`, I am receiving the error:
`Failed to update an existing Function's body on the Supabase project: {"message":"Internal Server Error","errorEventId":"[. . .]"}`
The project instance has many functions deployed. I am attempting to deploy changes to an existing function.

This error is applied to all functions when deploying. I checked the Supabase Server Status but everything reads as online. I also checked the AWS Server Status, which also reads everything as online.

I also tried logging in with a newly generated auth token.

**Comment by GaryAustin1** [comment, marked as answer]
There are a couple of reports in discord of similar issue starting today.
https://discord.com/channels/839993398554656828/843999948717555735
Edit:  And more here:  https://github.com/supabase/supabase/issues/7930
You probably should send a support ticket.

**Reply by WilliamScott-AdventGX** [reply]
Thank you for the heads up. Marking this as the answer as the community waits for support to investigate the issue.

I will also send them a report ticket. 👍
